index.html

```diff
@@ -98,6 +99,26 @@
 				maxime mollitia necessitatibus nobis pariatur quae quaerat quo. Doloremque, dolores eos necessitatibus tempora
 				tenetur vero.
 			</ts-note>
+			<article>
+				<h2>Pager</h2>
+				<ts-pager id="mypager" per-page="20" total-pages="10"></ts-pager>
+				<ts-pager per-page="20" total-pages="10" active-page="5"></ts-pager>
+				<ts-pager per-page="40" total-pages="10" active-page="1"></ts-pager>
+				<ts-pager total-pages="10" active-page="1"></ts-pager>
+				<ts-pager per-page="20" total-pages="14000" active-page="14000"></ts-pager>
+
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (10) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)